pkg/start/start.go

```diff
@@ -48,9 +48,9 @@ const (
 	//leaseDuration = 90 * time.Second
 	//renewDeadline = 45 * time.Second
 	//retryPeriod   = 30 * time.Second
```

Comment:
We should drop the TODO and these old commented-out values too.  Folks can blame the lines and see the commit message's detailed discussion to figure out why we have whatever values in the future.

Reply:
Fixed https://github.com/openshift/cluster-version-operator/commit/7328f5249d3535deb80f42f0e9e4765c8b581f4d